### PR TITLE
Add artsy open source footer

### PR DESCRIPTION
**Pull request by jonallured**
I'm doing another round of adding this footer to Artsy's open source repos.

### CODE REVIEW

**Comment by izakp** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -132,3 +132,28 @@ To create a new release, bump the version by editing the `./hokusai VERSION` fil
...
+Source by Default][footer_open] which means we strive to share as many details
+of our work as possible.
+
+You can learn more about this work from [our blog][footer_blog] and by following
+[@ArtsyOpenSource][footer_twitter] or explore our public data by checking out
+[our API][footer_api]. If you're interested in a career at Artsy, read through
+our [job postings][footer_jobs]!
+
+[footer_website]: https://www.artsy.net/
+[footer_principles]: culture/engineering-principles.md
```
Comment (izakp) [inline README.md]: Probably need absolute URLs here

Comment (jonallured) [inline README.md]:
```suggestion
[footer_principles]: https://github.com/artsy/README/blob/master/culture/engineering-principles.md
```

Comment (jonallured) [reply README.md]: Nice catch!

File: README.md [hunk 2]
```diff
@@ -132,3 +132,28 @@ To create a new release, bump the version by editing the `./hokusai VERSION` fil
...
+of our work as possible.
+
+You can learn more about this work from [our blog][footer_blog] and by following
+[@ArtsyOpenSource][footer_twitter] or explore our public data by checking out
+[our API][footer_api]. If you're interested in a career at Artsy, read through
+our [job postings][footer_jobs]!
+
+[footer_website]: https://www.artsy.net/
+[footer_principles]: https://github.com/artsy/README/blob/master/culture/engineering-principles.md
+[footer_open]: culture/engineering-principles.md#open-source-by-default
```
Comment (jonallured) [inline README.md]:
```suggestion
[footer_open]: https://github.com/artsy/README/blob/master/culture/engineering-principles.md
```

**Comment by jonallured** [comment]
Ok I fixed the couple of relative URLs so I'm going to merge this sucker - thanks @izakp!!

**Pull request merged**